tests/CatalogTests/PackageCatalogItemTests.cs

```diff
@@ -27,6 +27,11 @@ public class PackageCatalogItemTests
         [InlineData("InvalidDependencyVersionRange.0.1.0")] //A dependency with a version range that is invalid
         [InlineData("MissingDependencyVersionRange.0.1.0")] // A dependency with no version range attribute
         [InlineData("WhitespaceDependencyVersionRange.0.1.0")] // A dependency with a version range that is whitespace
+        [InlineData("PackageTypeCollapseDuplicate")]
```

Comment:
Same type different version?
Duplicate type name with different case?
Same type different version by System.Version but not SemVer (1.0, 1.0.0, 1.0.0.0)?
Whitespace around name? Whitepace around version?

It sucks we have to have the full nupkg and JSON data for each of these test variants. That's a lot of noise for small changes between tests.